### ISSUE TITLE
Example/template sixxsd configuration file

### ISSUE THREAD

**Issue by xelerance** [massar/sixxsd#2]
You wouldn't happen to have a sixxsd.conf file lying around, would you ?

**Comment by massar** [comment]
Please see the new commit that added: https://github.com/massar/sixxsd/blob/master/misc/sixxsd.conf

I'll note that pop_cmd_saveconfig() (See https://github.com/massar/sixxsd/blob/master/src/pop.c#L377 ) actually saves the configuration when it is running and contains a few details about the format, but indeed, that might not be so obvious, hence, I've added a commented configuration file there.

In that config and repeated in the README I've noted that sixxsd is really for historic purposes and also really requires a /40. See also the Security section of the README for more reasons to not use this code, next to that native IPv6 portion....


If this change satisfies your requirements, please then close this issue; or comment and we can discuss further ;)

**Comment by massar** [comment]
> We are Openswan. We got the security part figured out :-) But we just want to tinker with it.

Yes, with Openswan a few concerns would be solved, and with tinker I guess you likely mean learn from the techniques used, which is exactly why I opened the code up: so folks can learn from it.

If you have specific ideas/comments/questions, don't hesitate to ask/discuss!
